src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.js

```diff
@@ -140,16 +129,10 @@ const MetaList = () => {
     return routingDepartments ? options : options && [{ key: null, value: 'Niet gekoppeld' }, ...options];
   }, [categoryDepartments, routingDepartments]);
 
-
-  const handlingTime = useMemo(() => {
-    if (!incident?.category) return undefined;
-
-    const category = subcategoryOptions?.find(option => option.slug === incident.category.sub_slug);
-
-    if (!category) return undefined;
-
-    return getHandlingTime(category.sla.n_days, category.sla.use_calendar_days);
-  }, [incident, subcategoryOptions]);
+  const handlingTime = useMemo(
+    () => !incident?.category ? undefined : handlingTimesBySlug[incident.category.sub_slug],
```

Comment:
I would prefer a positive condition (i.e. without the exclamation mark).

Reply:
Sonar auto fixed my initial if statement through ESLint, I agree, there is an ESLint rule for this, and it is in the proposed rules I think, feel free to enable some ;)